### PR TITLE
V6 component link docs

### PR DESCRIPTION
**Pull request by dialpad-drew**
## Description
- Re-styles Link documentation
- Makes a couple style tweaks to the `codeWellHeader` and `codeWellFooter` shortcodes to smooth out some of the dark-background example styles
- Changes `--warning-color-focus` from `--yellow-200` to `yellow-400`. I think `--yellow-200` will be inaccessible and I didn't notice it until re-styling the page.
- Solves #206

### CODE REVIEW

File: lib/less/variables/colors.less [hunk 1]
```diff
@@ -195,7 +195,7 @@
     error-color-hsl:            var(--red-400-h) var(--red-400-s) var(--red-400-l);
     warning-color:              var(--yellow-500);
     warning-color-hover:        var(--yellow-600);
-    warning-color-focus:        var(--yellow-200);
+    warning-color-focus:        var(--yellow-400);
```
Comment (ryandialpad) [inline lib/less/variables/colors.less:198]: Is this change intended? I am guessing that the contrast ratio was not good enough

**Review by ryandialpad — APPROVED** [review]
w/ one question

Great work!

File: lib/less/variables/colors.less (hunk 1 above)
Comment (dialpad-drew) [reply lib/less/variables/colors.less:198]: Sure is and good guess! Explanation at the top of the PR.

Comment (ryandialpad) [reply lib/less/variables/colors.less:198]: 🤦 Thanks!

**Pull request merged**